query/lambdas/load_data/load_data.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+from query.lib.etl.transform import BundleDocumentTransform
+from query.lib.config import Config
+from query.lib.db.database import PostgresDatabase
+from query.lib.etl.extract import S3Extractor, Extractor
+from query.lib.etl.load import PostgresLoader, Loader
+from query.lib.etl.s3_client import S3Client
+from query.lib.common.logging import get_logger
+
+logger = get_logger('query.lambdas.load_data.load_data')
+
+# TODO: don't hardcode to staging, don't read directly from s3
+extractor = S3Extractor(S3Client('us-east-1', 'org-hca-dss-staging'))
+loader = PostgresLoader(PostgresDatabase(Config.serve_database_uri))
+
+
+class LoadData:
```

Comment:
1. Given that `LoadData` does not store any state and is just a container for the functions, you could either unpack the methods into the scope of the package/file or make them static. Since you've already created a file for these two methods that serves as a container, I'd just unpack them.
1. nit: If you choose to keep this class, the name `LoadData` sounds more like a verb than it does like a class object. Consider renaming it.

Reply:
👍